src/main/java/com/google/sps/servlets/GetTop3GemsServlet.java

```diff
@@ -26,19 +26,22 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.HiddenGem;
 
 /** Servlet that return the hidden gems dummy data. */
 @WebServlet("/recommendation")
 public class GetTop3GemsServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
     //TODO: For now, a hardcoded location is used. Later on, this servlet will probably be deleted as the filtering
     // based on the user's preferences will probably be done in the frontend. 
     LatLng hardcodedLocation = new LatLng(-33.8688, 151.2093);
-    Set<PlacesSearchResult> hiddenGems = Places.getAllHiddenGems(Places.getAllPlaces(hardcodedLocation));
+    Set<HiddenGem> hiddenGems = Places.getHiddenPlaces(Places.getAllHiddenGems(Places.getAllPlaces(hardcodedLocation)));
     Gson gson = new Gson();
-
+    
     // TODO: replace rankedHiddenGems with the recommendations from user's preference 
+    // TODO: remove this servlet.
```

Comment:
if it was done remove it, if not ignore this comment

Reply:
I am planning to do this in the other pr. Thanks for the reminder, I added a comment in the other PR to handle this :)